JumpscaleLibs/sal/reservation_chatflow/reservation_chatflow.py

```diff
@@ -631,31 +631,45 @@ def escrow_qr_show(self, bot, reservation_create_resp, expiration_provisioning):
         escrow_info = j.sal.zosv2.reservation_escrow_information_with_qrcodes(reservation_create_resp)
         escrow_address = escrow_info["escrow_address"]
         escrow_asset = escrow_info["escrow_asset"]
-        farmer_payments = escrow_info["farmer_payments"]
-        total_amount = escrow_info["total_amount"]
         reservationid = escrow_info["reservationid"]
         qrcode = escrow_info["qrcode"]
         remaning_time = j.data.time.secondsToHRDelta(expiration_provisioning - j.data.time.epoch)
+        payment_details = self.get_payment_details(escrow_info, escrow_asset.split(':')[0])
 
         message_text = f"""
-<h3> Please make your payment </h3>
-Scan the QR code with your application (do not change the message) or enter the information below manually and proceed with the payment. Make sure to add the reservationid as memo_text.
-<p>If no payment is made in {remaning_time} the reservation will be canceled</p>
-
-<h4> Escrow address: </h4>  {escrow_address} \n
-<h4> Escrow asset: </h4>  {escrow_asset} \n
-<h4> Total amount: </h4>  {total_amount} \n
-<h4> Reservation id: </h4>  {reservationid} \n
-
-<h4>Payment details:</h4> \n
-"""
-        for payment in farmer_payments:
-            message_text += f"""
-Farmer id : {payment['farmer_id']} , Amount :{payment['total_amount']}
-"""
+        <h3> Please make your payment </h3>
+        Scan the QR code with your application (do not change the message) or enter the information below manually and proceed with the payment. Make sure to add the reservationid as memo_text.
+        <p>If no payment is made in {remaning_time} the reservation will be canceled</p>
+
+        <h4> Escrow address: </h4>  {escrow_address} \n
+        <h4> Escrow asset: </h4>  {escrow_asset} \n
+        <h4> Reservation id: </h4>  {reservationid} \n
+        <h4> Payment details: </h4> {payment_details} \n
+        """
 
         bot.qrcode_show(data=qrcode, msg=message_text, scale=4, update=True, html=True)
 
+    def get_payment_details(self, escrow_info, currency):
+
+        farmer_payments = escrow_info["farmer_payments"]
+        total_amount = escrow_info["total_amount"]
+
+        explorer = j.clients.explorer.default
+        payment_details = ""
+        if currency == 'FreeTFT':
+           payment_details += f"<tr><td>Transaction Fees</td><td>{0.1} {currency}</td></tr>"
```

Comment:
Why do you have a different case for FreeTFT ?

Reply:
According to this payment_distribution, the FreeTFT has only 0.1 transaction fees
https://github.com/threefoldtech/tfexplorer/blob/master/pkg/escrow/payout_distribution.go

Reply:
Not sure how you deduct this from the code you linked.
But in any case, in the else block you already hardcode `Transaction Fees</td><td>{0.1} `
The only difference with FreeTFT than other currency is that the tokens are burned and not sent to the farmer.
This also means the foundation doesn't get anything out of it.

Reply:
so in the FreeTFT case the burned amount should be shown?